src/transformer.ts

```diff
@@ -6,7 +6,7 @@ export class EncryptionTransformer implements ValueTransformer {
   constructor(private options: EncryptionOptions) {}
 
   public from(value?: string | null): string | undefined {
-    if (value === null || value === undefined) {
+    if (!value) {
```

Comment:
This is called when `find` or `select` is called. This is fine since an empty string being read from the database isn't possible unless it was NULL.